pkg/kubernetes/pkg/util/metrics/metrics.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
```

Comment:
This is not an appropriate place for this file to exist. All files under `pkg/kubernetes` are copied from kube/kube. See https://github.com/kubernetes-incubator/service-catalog/blob/master/pkg/kubernetes/README.md.